source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -84,8 +84,7 @@ void renderBackgroundImage(const std::shared_ptr<AdaptiveCards::BackgroundImage>
 
         if (imgView) {
             imgView.translatesAutoresizingMaskIntoConstraints = NO;
-            [containerView addSubview:imgView];
-            [containerView sendSubviewToBack:imgView];
+            [containerView insertSubview:imgView atIndex:0];
```

Comment:
>containerView [](start = 13, length = 13)

What happens today? And what will happen after this change, to the current view at index0 that may already be there?

Reply:
This is merely cosmetic. the codes do essentially same. 

---
In reply to: [387940578](https://github.com/microsoft/AdaptiveCards/pull/3817#discussion_r387940578) [](ancestors = 387940578)